src/System.Private.DataContractSerialization/src/System.Private.DataContractSerialization.csproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <AssemblyName>System.Private.DataContractSerialization</AssemblyName>
-    <AssemblyVersion>4.1.1.0</AssemblyVersion>
+    <AssemblyVersion>4.1.1.1</AssemblyVersion>
```

Comment:
It used to be the case where the package version was matching the assembly version. however CoreFX has moved to another scheme where all the packages are being built with the same package version and the assembly version revs independently.